Unable to build library using grunt. Fatal error: Protocol error (Runtime.callFunctionOn): Object reference chain is too long.

<!--
Hi there!

To check any option, replace the "[ ]" with a "[x]". Be sure to check out how it looks in the Preview tab! Feel free to remove any portion of the template that is not relevant for your issue.
-->

#### Most appropriate sub-area of p5.js?

- [ ] Accessibility (Web Accessibility)
- [x] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Details about the bug:
Unable to build library using grunt

- p5.js version: 1.4.0
- Web browser and version: Google Chrome Version 92.0.4515.107 (Official Build) (64-bit)
- Operating System: Windows 10 Build 19043.1110
- Node version: 16.5.0 
- Steps to reproduce this:
```bash
npm run grunt
```

Console error:
```bash
Fatal error: Protocol error (Runtime.callFunctionOn): Object reference chain is too long
```

I couldn't do a straightforward update to puppeteer 10.2.0 (latest version) because of a few failed unit tests that I'm not too familiar with. @outofambit Can you perhaps have a look?

```
  1) Global Error Handling
       identifies TypeError 'readFromNull':

      AssertionError: expected 0 to equal 1
      + expected - actual

      -0
      +1
      
      at Function.assert.strictEqual (http://localhost:9001/node_modules/chai/chai.js:2329:32)
      at http://localhost:9001/test/unit/core/error_helpers.js:774:14

  2) Global Error Handling
       identifies TypeError 'readFromUndefined':

      AssertionError: expected 0 to equal 1
      + expected - actual

      -0
      +1
      
      at Function.assert.strictEqual (http://localhost:9001/node_modules/chai/chai.js:2329:32)
      at http://localhost:9001/test/unit/core/error_helpers.js:792:14

  3) time and date
       p5.prototype.millis
         result should be greater than running time:
     AssertionError: everything is ok: expected 50 to be > 50
```

--- a/src/core/friendly_errors/browser_errors.js
+++ b/src/core/friendly_errors/browser_errors.js
@@ -91,7 +91,7 @@ const strings = {
       browser: 'all'
     },
     {
-      msg: 'Cannot read {{.}} of null',
+      msg: 'Cannot read {{.}} null',
       type: 'READNULL',
       browser: 'Chrome'
     },

--- a/src/core/friendly_errors/browser_errors.js
+++ b/src/core/friendly_errors/browser_errors.js
@@ -91,7 +91,7 @@ const strings = {
       browser: 'all'
     },
     {
-      msg: 'Cannot read properties of null',
+      msg: 'Cannot read {{.}} of null',
       type: 'READNULL',
       browser: 'Chrome'
     },
@@ -101,7 +101,7 @@ const strings = {
       browser: 'Firefox'
     },
     {
-      msg: 'Cannot read properties of undefined',
+      msg: 'Cannot read {{.}} undefined',
       type: 'READUDEFINED',
       browser: 'Chrome'
     },

--- a/src/core/friendly_errors/browser_errors.js
+++ b/src/core/friendly_errors/browser_errors.js
@@ -91,7 +91,7 @@ const strings = {
       browser: 'all'
     },
     {
-      msg: "Cannot read property '{{.}}' of null",
+      msg: 'Cannot read property of null',
       type: 'READNULL',
       browser: 'Chrome'
     },
@@ -101,7 +101,7 @@ const strings = {
       browser: 'Firefox'
     },
     {
-      msg: "Cannot read property '{{.}}' of undefined",
+      msg: 'Cannot read property of undefined',
       type: 'READUDEFINED',
       browser: 'Chrome'
     },

--- a/src/core/friendly_errors/browser_errors.js
+++ b/src/core/friendly_errors/browser_errors.js
@@ -91,7 +91,7 @@ const strings = {
       browser: 'all'
     },
     {
-      msg: 'Cannot read property of null',
+      msg: 'Cannot read properties of null',
       type: 'READNULL',
       browser: 'Chrome'
     },
@@ -101,7 +101,7 @@ const strings = {
       browser: 'Firefox'
     },
     {
-      msg: 'Cannot read property of undefined',
+      msg: 'Cannot read properties of undefined',
       type: 'READUDEFINED',
       browser: 'Chrome'
     },

--- a/test/unit/utilities/time_date.js
+++ b/test/unit/utilities/time_date.js
@@ -110,7 +110,7 @@ suite('time and date', function() {
       var runningTime = 50;
       var init_date = window.performance.now();
       // wait :\
-      while (window.performance.now() - init_date < runningTime) {
+      while (window.performance.now() - init_date <= runningTime) {
         /* no-op */
       }
       assert.operator(myp5.millis(), '>', runningTime, 'everything is ok');
@@ -121,7 +121,7 @@ suite('time and date', function() {
       var init_date = Date.now();
       var result = myp5.millis();
       // wait :\
-      while (Date.now() - init_date < runningTime) {
+      while (Date.now() - init_date <= runningTime) {
         /* no-op */
       }
       var newResult = myp5.millis();

